reproducibility_project/src/engines/gomc/project.py

```diff
@@ -576,24 +578,25 @@ def part_3d_output_production_run_started(job):
 # function for checking if GOMC simulations are completed properly
 def gomc_sim_completed_properly(job, control_filename_str):
     """General check to see if the gomc simulation was completed properly."""
-    job_run_properly_bool = False
-    output_log_file = "out_{}.dat".format(control_filename_str)
-    if job.isfile(output_log_file):
-        # with open(f"workspace/{job.id}/{output_log_file}", "r") as fp:
-        with open(f"{output_log_file}", "r") as fp:
-            out_gomc = fp.readlines()
-            for i, line in enumerate(out_gomc):
-                if "Move" in line:
-                    split_move_line = line.split()
-                    if (
-                        split_move_line[0] == "Move"
-                        and split_move_line[1] == "Type"
-                        and split_move_line[2] == "Mol."
-                        and split_move_line[3] == "Kind"
-                    ):
-                        job_run_properly_bool = True
-    else:
+    with job:
```

Comment:
Why do we need this with job statement here?

Reply:
With statement is needed because when this test is used with aggregator, test doesn't see the file it's supposed to check.